### ISSUE TITLE
add more locale

### ISSUE THREAD

**Issue by tomasiek** [alllexx88/Optware-ng#15]
Is it possible to add the locale pl_PL.UTF-8 ?
I have a problem with special characters (like ąęśćżńł) in Midnight commaner and own scripts.
I'm trying to replace entware with optware-ng on linksys e3200 with tomatoshibby. Polish special characters is the only thing I was missing

**Comment by alllexx88** [comment]
It is doable, but in uClibc locale data is compiled into the library, unlike with glibc. This helps to save space, but means you can't add/remove locale during runtime. I just tried to compile uClibc-ng with all the locales, and it is approximately now twice as large as before. It's fine, though, since I don't think that additional ~ 1MB is really an issue in the modern days, don't you agree?

**Comment by tomasiek** [comment]
It's fine for me...How can I test it?
~ 1MB - it makes no difference for me :)

**Comment by alllexx88** [comment]
I've just uploaded the upgrade. Simply run:

```
ipkg update
ipkg upgrade
```

**Comment by tomasiek** [comment]
ok, I will try it tonight. thx.

2015-11-10 18:58 GMT+01:00 alllexx88 <notifications@github.com>:

> I've just uploaded the upgrade. Simply run:
>
> ipkg update
> ipkg upgrade
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alllexx88/Optware-ng/issues/15#issuecomment-155514158>
> .
>



-- 
Pozdrawiam
        Tomasz Woś

**Comment by tomasiek** [comment]
Works great - there is small problem with fonts in dialog, but mc/nano works great.
btw libjpeg is very (6)old and it's not compatibile with minidlna-rescan (requires 8) :(

**Comment by alllexx88** [comment]
Good. As for `jpeg`, there was a packaging bug, thanks for noticing. Fixed, and will update the feeds soon. For now, you can upgrade it manually:
http://optware-ng.zyxmon.org/buildroot-armeabi-ng/libjpeg_8d-1_arm.ipk

**Comment by alllexx88** [comment]
Ah, sorry, you router is MIPSEL:
http://optware-ng.zyxmon.org/buildroot-mipsel-ng/libjpeg_8d-1_arm.ipk

**Comment by alllexx88** [comment]
OK, feeds updated.

**Comment by tomasiek** [comment]
It works everything. As for the PL locale, I noticed a problem with a package "dialog". Do not display Polish characters, do not display them properly when typing, but when I view the result by the echo - they are displayed properly. It's a small problem. I suspect that there is something wrong with utf-8 in this package.
Minidlna now works perfectly. The fast rescan option is great.
BTW, where i can post for adding some new packages (I used earlier Entware and a few I'm missing)
